server/handler/base.go

```diff
@@ -111,6 +122,87 @@ func (b *Base) postGitHubRepoStatus(ctx context.Context, client *github.Client,
 	return err
 }
 
+func (b *Base) PostCheckResults(ctx context.Context, client *github.Client, pr *github.PullRequest, evaluationResult common.Result) error {
+	sha := pr.GetHead().GetSHA()
+	owner := pr.GetBase().GetRepo().GetOwner().GetLogin()
+	repo := pr.GetBase().GetRepo().GetName()
+
+	for _, result := range evaluationResult.Children {
+		if err := b.createGitHubRepoCheck(ctx, client, pr, owner, repo, sha, *result); err != nil {
```

Comment:
This ends up creating both approval and disapproval runs, right? Would it be better to combine them into a single run or does the fact that they are part of the same suite effectively combine them?

Reply:
That's correct. In my mind it more clearly splits out what has been the failed checks and the checks that have not. However, as they are both part of the same suite, which is tied to a ref, it shouldn't really matter underneath